lib/sequenceserver/blast/job.rb

```diff
@@ -24,13 +24,23 @@ def initialize(params)
             @databases = Database[params[:databases]]
             @advanced  = params[:advanced].to_s.strip
             @options   = @advanced + defaults
+            @num_threads = config[:num_threads]
+            @query_length = query_size
+            @total_database_length = ncharacters_total
```

Comment:
As mentioned earlier, it is not clear if database_length means number of sequences or number of characters. A better name is `@databases_ncharacters_total`.

Reply:
Ok I forgot that, I'll update that.